devtools/conda_install_reqs.sh

```diff
@@ -9,12 +9,23 @@
 
 DEVTOOLS_DIR=`dirname "${BASH_SOURCE[0]}"`
 
+if ! command -v mamba &> /dev/null
+then
+    EXE="conda"
+else
+    EXE="mamba"
+fi
+
+INSTALL_CMD="$EXE install -y -q -c conda-forge -c omnia --override-channels"
+
+# TODO: is this preinstall needed? We're certainly no longer using pyyaml,
+# and it looks like we should be, but aren't, using future in setup_cfg_reqs
 if [ ! -z "$OPS_ENV" ]
 then
-    conda create -q -y --name $OPS_ENV conda future pyyaml python=$CONDA_PY
+    $INSTALL_CMD --name $OPS_ENV conda future pyyaml python=$CONDA_PY
```

Comment:
does conda needs to be installed here?
```suggestion
    $INSTALL_CMD --name $OPS_ENV future pyyaml python=$CONDA_PY
```